test/integration/coronavirus_local_restrictions_test.rb

```diff
@@ -197,11 +197,11 @@ class CoronavirusLocalRestrictionsTest < ActionDispatch::IntegrationTest
       then_i_see_details_of_christmas_rules
     end
 
-    it "does not display christmas rules for future tier four restrictions" do
+    it "displays christmas rules for future tier four restrictions" do
```

Comment:
I don't understand. I thought Christmas rules never apply to a place going into tier 4?

Reply:
We should now because we know that no new tier 4 restrictions are coming into effect before Christmas. In its current state it won't show the Christmas rules to someone currently in Tier 3 going into Tier 4 after Christmas.

Reply:
Ok. That makes sense.

Reply:
nice one, thanks :)